probe/probe.go

```diff
@@ -582,6 +589,57 @@ func getAnnotatedPodLabels(m map[string]string) mapset.Set[string] {
 	return b
 }
 
+// Store container default posture data
+func setPostureData(buf []byte) error {
+	var kd *KubeArmorProbeData
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
+	err := json.Unmarshal(buf, &kd)
```

Comment:
We already unmarshal once, can we extract and pass on the data to `getSecurityPostureAndVisibility` directly from there eliminating the need to redo unmarshling also removes the global variable. Wdyt?

Reply:
Ya, we can do that, I think it would require some changes in `readDataFromKubeArmor`  and  `printKubeArmorProbeOutput` function